src/Text-Core/RunArray.class.st

```diff
@@ -737,18 +680,17 @@ RunArray >> setRuns: newRuns setValues: newValues [
 
 { #category : #accessing }
 RunArray >> size [
-	| size |
-	size := 0.
-	1 to: runs size do: [:i | size := size + (runs at: i)].
-	^size
+
+	^ runs sumNumbers
```

Comment:
here also, the initial implementation is faster because of inlining, and `size` might be called a lot.
Maybe the impact is minimal but without perf measures, I would prefer to keep the original code. better be safe.

Reply:
I'll add to my todo to check that. I'll add a Work Needed label in the meantime